src/Jobs/ContentReviewNotificationJob.php

```diff
@@ -76,7 +84,7 @@ public function process()
         $this->queueNextRun();
 
         $task = new ContentReviewEmails();
-        $task->run(new SS_HTTPRequest("GET", "/dev/tasks/ContentReviewEmails"));
+        $task->run(new HTTPRequest("GET", "/dev/tasks/ContentReviewEmails"));
 
         $this->currentStep = 1;
         $this->isComplete = true;
```

Comment:
On line 109 below, replace the string literal with it's static class equivalent.